test/cmd/image.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+source "$(dirname "${BASH_SOURCE}")/../../hack/lib/init.sh"
+trap os::test::junit::reconcile_output EXIT
+
+# Cleanup cluster resources created by this test
+(
+  set +e
+  oc delete all --all
+  exit 0
+) &>/dev/null
+
+
+os::test::junit::declare_suite_start "cmd/registry/login"
+os::cmd::expect_success_and_text "oc registry login -z 'default' --to=- --skip-check" "auth"
+os::cmd::expect_failure_and_text "oc registry login -z 'default' --registry=localhost2 --to=- 2>&1" "unable to check your credentials"
+os::cmd::expect_success_and_text "oc registry login -z 'default' --registry=localhost2 --to=/tmp/test --skip-check && cat /tmp/test" "localhost2"
+os::test::junit::declare_suite_end
+
+os::test::junit::declare_suite_start "cmd/registry/info"
+os::cmd::expect_success "oc registry info"
+os::cmd::expect_failure_and_text "oc registry info --internal --public" "only one of --internal or --public"
```

Comment:
I'd expect a bit more tests, to be honest ;)

Reply:
These are all in a different PR, but registry info is difficult to test with our existing framework because it requires master reconfig

Reply:
Comment on https://github.com/openshift/origin/pull/18930